File: thirtyfive.py
# thirtyfive.py by Jacob Steinebronn
# The idea of the solution is 6*s + o

# Take in number of test cases
numTests = (int)(input().strip())

for testNum in range(numTests):
    # Read in s, o and make sure they're of type int
    s, o = list(map(int, input().strip().split()))

    # The answer is s*base^1 + o*base^0
    # So if the base is 6, then the answer is
    # 6s + o
    print(6 * s + o)

File: thirtyfive.java

import java.util.Scanner;

public class thirtyfive
{
   public static void main(String[] args) 
   {
      // Read in the number of cases
      Scanner scan = new Scanner(System.in);
      int t = scan.nextInt();

      // Loop through the cases
      for (int i = 0; i < t; i++) 
      {
         // Get the two digits
         int s = scan.nextInt();
         int o = scan.nextInt();

         // Output the answer (s * 6 + o)
         System.out.println(s * 6 + o);
      }
   }
}

